src/utils/webrtc/simplewebrtc/simplewebrtc.js

```diff
@@ -118,6 +118,12 @@ function SimpleWebRTC(opts) {
 				} else {
 					self.emit('mute', { id: message.payload.peerId })
 				}
+			} else if (message.payload.action === 'raiseHand') {
+				if (message.from === self.connection.getSessionId()) {
```

Comment:
Since we don't have the peer id here, I saw that "from" seems to contain an id. But it feels wrong as it didn't seem to map to peers.

This is different than "forceMute" as it's one participant sending to everyone else, not one moderator broadcasting to all with a specific peer id to mute.

@danxuliu any advice here ?